docs/Releasing-the-plugin.md

```diff
@@ -0,0 +1,25 @@
+# Releasing the performance plugin
+
+This document describes the steps to release the Performance plugin.
+
+## Preparing the release
+
+> The following changes need to be made through a pull request.
+
+### Update the version number
+
+The version number needs to be updated in the following files:
+
+- package.json
+- load.php
+- readme.txt
+
+### Update readme.txt
+
+Run `npm run readme -- -m "X.Y.Z"` to update the readme.txt file with the release changelog, with X.Y.Z as the release milestone name.
+
+## Create a new GitHub release
+
+Go to https://github.com/WordPress/performance/releases/new to create a new release for the plugin. The release tag should be in the format `X.Y.Z`. Finally, add the changelog (it can be found in the readme.txt file) to the release description and create the release.
```

Comment:
Maybe elaborate a bit, for example it would be good to specify the branch to use and maybe also to inform the other plugin maintainers to not add any commits or merge any more PRs at this time.

```suggestion
Once the above pull request has been merged, let the other maintainers know on [Slack](https://wordpress.slack.com/archives/performance) that no new commits or pull requests must be added to the branch due to the release process. Then, make sure that all GitHub actions successfully pass for the target branch (e.g. [for `trunk`](https://github.com/WordPress/performance/actions?query=branch%3Atrunk)).

After that, [create a new release tag for the plugin on GitHub](https://github.com/WordPress/performance/releases/new). The release tag should have the same name as the corresponding milestone used earlier, and it should be created from the `trunk` branch (unless this is for a patch release, in which case it should be created from the corresponding `release/...` branch). Finally, add the changelog (it can be found in the readme.txt file) to the release description and create the release.
```